modules/xmpp/JingleSessionPC.js

```diff
@@ -2015,20 +2036,25 @@ export default class JingleSessionPC extends JingleSession {
      * Adds a new track to the peerconnection. This method needs to be called only when a secondary JitsiLocalTrack is
      * being added to the peerconnection for the first time.
      *
-     * @param {JitsiLocalTrack} localTrack track to be added to the peer connection.
+     * @param {Array<JitsiLocalTrack>} localTracks - Tracks to be added to the peer connection.
      * @returns {Promise<void>} that resolves when the track is successfully added to the peerconnection, rejected
      * otherwise.
      */
-    addTrack(localTrack) {
-        if (!FeatureFlags.isMultiStreamSupportEnabled() || localTrack.type !== MediaType.VIDEO) {
-            return Promise.reject(new Error('Multiple tracks of a given media type are not supported'));
+    addTracks(localTracks = null) {
```

Comment:
we need to add an adapter and keep the `addTrack` method (mark as deprecated) as it's an API breaking change, something that will do `addTracks([ track])` internally and fail for the multi stream I guess? throw an error that says use addTracks in multi stream, WDYT?

Reply:
This is the new method that we added only for multi-stream support at JingleSessionPC level. This wouldn't be considered a breaking change.

Reply:
Oh got it, I thought it was JitsiConference